src/OperationProcessor.ts

```diff
@@ -209,12 +215,13 @@ class OperationProcessorImpl implements OperationProcessor {
    *            1. The operation (of the same hash) is not process before; or
    *            2. The operation is processed before but this operation has an earlier timestamp.
    *          Returns undefined if the same operation with an earlier timestamp was processed previously.
+   *
```

Comment:
I am not so sure. It is still returning the opHash - like we discussed long back it is true that the opHash is not used by the observer but the tests still use it like before. 

But now that the unit tests know how to hash operations, we have the option of removing the opHash return. Is this what you are suggesting?

Reply:
It is fine to return the operation hash if tests still uses it, but the comments still appear obsolete? E.g, the method doesn't return undefined like it claims it could.

Reply:
yeah - makes sense. I fixed the signature to return Promise<string> - changed the OperationProcessor interface as well.